javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -336,6 +336,17 @@ public static boolean isBooleanType(TypeMirror type) {
         return isDeclaredOfName(type, "java.lang.Boolean") || type.getKind() == TypeKind.BOOLEAN;
     }
 
+    /**
+     * Checks if the type represents a character type, that is either char (primitive type) or
```

Comment:
I see that this is similar to the comment on the above method, but shouldn't it be `that is, is either...`? At the moment, doesn't this mean that there are other possible options for "character type", but we return true only for these two?

Reply:
I thought this means we are checking if the given `TypeMirror` is a boxed or primitive character type. Could you please elaborate on "other possible options"?